### PR TITLE
RTX5 irq_armv7m.s forgot to cover FPv5_D16 and FPv5_SP_D16

### PR DESCRIPTION
**Pull request by GorgonMeducer**
I am not the first person who found this issue:

I observed the issue in a Chinese bbs:
http://www.armbbs.cn/forum.php?mod=viewthread&tid=108052&highlight=CMSIS

The first reporter is eric2013, who is the owner of this bbs.
Since I have confirmation that he doesn't have a GitHub account, and he explicitly suggested that the arm open-source community should fix this, I create this patch.

### CODE REVIEW

**Comment by grasci-arm** [comment]
Can one of the admins verify this patch?

File: CMSIS/RTOS2/RTX/Source/ARM/irq_armv7m.s [hunk 1]
```diff
@@ -28,7 +28,7 @@
 RTX_STACK_CHECK EQU      0
                 ENDIF
 
-                IF       ({FPU}="FPv4-SP")
+                IF       ({FPU}="FPv4-SP") || ({FPU}="FPv5_D16") || ({FPU}="FPv5_SP_D16")
```
Comment (JonatanAntoni) [inline CMSIS/RTOS2/RTX/Source/ARM/irq_armv7m.s:31]: Please double check with #1266.

**Review by JonatanAntoni — CHANGES_REQUESTED** [review]
Thanks for raising this as a PR.
This issue is already discussed in #1266.

**Comment by JonatanAntoni** [comment]
@GorgonMeducer, may I ask you to properly rebase your pull-request, please? I'd expect it only lists a single commit introducing the fix. Merge commits easily cause issues when merging the PR.

**Comment by GorgonMeducer** [comment]
> @GorgonMeducer, may I ask you to properly rebase your pull-request, please? I'd expect it only lists a single commit introducing the fix. Merge commits easily cause issues when merging the PR.

OK, will do that next time.

**Pull request closed without merging**